examples/bt-get2.cpp

```diff
@@ -85,7 +85,7 @@ int main(int argc, char const* argv[])
 		, std::istream_iterator<char>()};
 
 	lt::error_code ec;
-	lt::add_torrent_params atp = lt::read_resume_data(&buf[0], buf.size(), ec);
+	lt::add_torrent_params atp = lt::read_resume_data(&buf[0], int(buf.size()), ec);
```

Comment:
@glassez it was made by intention: https://github.com/arvidn/libtorrent/pull/624#discussion_r59974899
"type()" form was chosen, so any further pull requests use such style